dp-hello-world-event/event/handler_test.go

```diff
@@ -3,16 +3,19 @@ package event_test
 import (
 	"github.com/ONSdigital/dp-hello-world-event/event"
 	. "github.com/smartystreets/goconvey/convey"
+	"io/ioutil"
 	"testing"
 )
 
 func TestHelloCalledHandler_Handle(t *testing.T) {
 
-	Convey("Given a successful event handler, when Handle is triggered", t, func() {
+	Convey("Given a successful event handler, when Handle is triggered it writes the message to a file", t, func() {
 
 		eventHandler := event.HelloCalledHandler{}
 		err := eventHandler.Handle(testCtx, &testEvent)
 		So(err, ShouldBeNil)
+		data, err := ioutil.ReadFile("/tmp/helloworld.txt")
```

Comment:
Could the file remain from a previous run? What if the code is changed to remove saving to the temp file but the test succeeds only because the file was already there? Maybe if the event contained a unique code or something that this test checked was in the file after we would now it was written by this test?

Reply:
wanted to make a few and as simple changes as poss, but good points to address, will review this later.